projects/ghostscript/Dockerfile

```diff
@@ -16,13 +16,21 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 
-RUN apt-get update && apt-get install -y autoconf zlibc libtool liblcms2-dev libpng-dev
+RUN apt-get update && apt-get install -y autoconf zlibc libtool liblcms2-dev libpng-dev wget
 RUN git clone --branch branch-2.2 --single-branch --depth 1 https://github.com/apple/cups.git cups
 RUN git clone --branch VER-2-12-1 --single-branch --depth 1 https://git.savannah.gnu.org/git/freetype/freetype2.git freetype
 RUN git clone --single-branch --depth 1 git://git.ghostscript.com/ghostpdl.git ghostpdl
 RUN mkdir ghostpdl/fuzz
 
+# Get PDF seeds
+RUN mkdir $SRC/pdf_seeds && \
+    cd $SRC/pdf_seeds && \
+    wget https://github.com/MozillaSecurity/fuzzdata/raw/master/samples/pdf/pdf.pdf
```

Comment:
Do we really want to do this everytime?  If we have adequate rights to the pdf, can we just add it to the repo?

Reply:
It won't be downloaded a lot of times as it'll only be downloaded each time the container is created. I'm happy to leave it as is and would prefer to leave it in the Mozilla repo than having it in the OSS-Fuzz repo -- are you okay with this?

Reply:
I would rather copy it and not assume that the other repo won't change and have something outside of the codebase break the fuzzer.